README.md

```diff
@@ -0,0 +1,100 @@
+uData
+=====
+
+[![Build status](https://secure.travis-ci.org/etalab/udata.png)](https://coveralls.io/repos/etalab/udata/badge.png?branch=master)
+[![Code coverage](https://coveralls.io/repos/etalab/udata/badge.png?branch=master)](https://coveralls.io/repos/etalab/udata/badge.png?branch=master)
+[![Python Requirements Status](https://requires.io/github/etalab/udata/requirements.png?branch=master)](https://requires.io/github/etalab/udata/requirements/?branch=master)
+[![JavaScript Dependencies Status](https://david-dm.org/etalab/udata.svg)](https://david-dm.org/etalab/udata)
+[![Join the chat at https://gitter.im/etalab/udata](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/etalab/udata?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+_uData_ is an open and social data hub.
+
+
+Compatibility
+-------------
+
+udata requires Python 2.7, MongoDB 3.2+, ElasticSearch 1.7 and Redis.
+
+
+Installation
+------------
+
+### System dependencies
+
+sudo apt-get install -y --no-install-recommends tar git wget curl build-essential pkg-config  python python-dev python-pip libjpeg-dev zlib1g-dev libpng12-dev libtiff5-dev libfreetype6-dev liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev libxml2-dev libxslt1-dev liblzma-dev libyaml-dev && sudo pip install virtualenv cython
+
+> MacOSX: checkout Homebrew for installing dependencies.
+
+
+### Installing the project
+
+Let's call the root project `$WORKSPACE`.
+
+    mkdir $WORKSPACE && cd $_
+
+Retrieve the sources
+
+    git clone https://github.com/etalab/udata.git
+
+Install your [virtualenv](https://virtualenv.readthedocs.org/en/latest/)
```

Comment:
It depends on if we use docker-compose for the stack services only (Mongo, Elasticsearch, Redis...) or the development environment too. Right now, we are using docker-compose only for services, not the complete environment.